source/index.txt

```diff
@@ -7,18 +7,16 @@ Cluster-to-Cluster Sync
 .. default-domain:: mongodb
 
 {+c2c-product-name+} provides continuous data synchronization or a 
-one-time data migration between MongoDB clusters. You can enable
-{+c2c-product-name+} with the :ref:`mongosync <c2c-mongosync>` utility.
+one-time data migration between MongoDB clusters. For current considerations on 
+``mongosync`` usage, see the :ref:`<mongosync-considerations>` page. You can 
```

Comment:
Suggest rephrasing to, "For notes and caveats on continuous sync, see the ___ page"